release/microbenchmark/experimental/accelerated_dag_gpu_microbenchmark.py

```diff
@@ -58,13 +58,16 @@ class TorchTensorWorker:
     def __init__(self):
         self.device = torch_utils.get_devices()[0]
 
-    def send(self, shape, dtype, value: int):
-        t = torch.ones(shape, dtype=dtype, device=self.device) * value
+    def send(self, shape, dtype, _):
+        t = torch.ones(shape, dtype=dtype, device=self.device) * 1
```

Comment:
In pure NCCl benchmark `exec_nccl_gpu`, the function `do_send_recv` also multiplies a tensor.